src/renderer/components/ScreenSharePicker.tsx

```diff
@@ -424,7 +424,7 @@ function ModalComponent({
                         }
                         console.log([...MediaEngineStore.getMediaEngine().connections]);
                         try {
-                            if (conn.streamUserId === UserStore.getCurrentUser().id) {
+                            if (conn) {
```

Comment:
```suggestion
                            if (conn?.streamUserId === UserStore.getCurrentUser().id) {
```

Reply:
Do we really need this? Because conn variable is exists only when we previously find it by comparing IDs:
```
var conn = [...MediaEngineStore.getMediaEngine().connections].find(
  connection => connection.streamUserId === UserStore.getCurrentUser().id
);
```

Reply:
With how it's implemented currently it is needed as conn is defined within a try-catch. If it were to raise we wouldn't have `conn`

Reply:
Maybe we can just write `if(conn?)`? I just want to not do unnecessary === calls

Reply:
Ah you're right as conn will already be defined as the user's stream. Yeah I think just using `if (conn)` would be enough